controllers/cloudflareCache.js

```diff
@@ -53,7 +53,7 @@ const purge = async (res, id, username, isSuperUser, userDetails) => {
         await logsQuery.addLog(
           logType.CLOUDFLARE_CACHE_PURGED,
           { userId: id, purgedFor: userDetails.user.id },
-          { message: "Cache Purged" }
+          { message: "Cache Purged" },
```

Comment:
do we really need this ","

Reply:
It's Due to new prettier update, prettier is adding that , while fixing the linting error, else it's throwing linting error @iamitprakash

Reply:
No, it is not needed as such as prettier is already covering that rule, Made the changes can you review this again. @iamitprakash.